tests/network-tests/src/Api.ts

```diff
@@ -1170,25 +1178,26 @@ export class Api {
     videoId: number,
     auctionParams?: OpenAuctionParams | EnglishAuctionParams
   ): Promise<ISubmittableResult> {
-    const initTransactionalStatus = this.api.createType(
-      'InitTransactionalStatus',
+    const initTransactionalStatus = createType(
+      'PalletContentNftTypesInitTransactionalStatusRecord',
       auctionParams
-        ? auctionParams instanceof OpenAuctionParams
-          ? { OpenAuction: auctionParams }
-          : { EnglishAuction: auctionParams }
+        ? // eslint-disable-next-line no-prototype-builtins
+          auctionParams.hasOwnProperty('bidLockDuration')
+          ? { OpenAuction: auctionParams as OpenAuctionParams }
+          : { EnglishAuction: auctionParams as EnglishAuctionParams }
         : { Idle: null }
     )
 
-    const updateParameters = this.createType('VideoUpdateParameters', {
-      assets_to_upload: null,
-      new_meta: null,
-      assets_to_remove: this.api.createType('BTreeSet<DataObjectId>', []),
-      enable_comments: false,
-      auto_issue_nft: this.api.createType('NftIssuanceParameters', {
+    const updateParameters = createType('PalletContentVideoUpdateParametersRecord', {
+      assetsToUpload: null,
+      newMeta: null,
+      assetsToRemove: [],
+      expectedDataObjectStateBloatBond: 0,
```

Comment:
Shouldn't be hardcoded

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)